Providers/Modules/nxOMSAuditdPlugin/conf/auditd_plugin.conf

```diff
@@ -0,0 +1,29 @@
+<source>
+  @type auoms
```

Comment:
Interesting. Our conf files seem to all use `type`, but you've tested this works I suppose.